geode-core/src/main/java/org/apache/geode/management/internal/operation/RebalanceResultImpl.java

```diff
@@ -21,16 +21,110 @@
 import org.apache.geode.management.runtime.RebalanceResult;
 
 public class RebalanceResultImpl implements RebalanceResult {
-  private Map<String, Map<String, Long>> rebalanceSummary = new LinkedHashMap<>();
+  private Map<String, PerRegionStats> rebalanceSummary = new LinkedHashMap<>();
 
   RebalanceResultImpl() {}
 
   @Override
-  public Map<String, Map<String, Long>> getRebalanceSummary() {
+  public Map<String, PerRegionStats> getRebalanceStats() {
     return rebalanceSummary;
   }
 
-  void setRebalanceSummary(Map<String, Map<String, Long>> rebalanceSummary) {
+  void setRebalanceSummary(Map<String, PerRegionStats> rebalanceSummary) {
     this.rebalanceSummary = rebalanceSummary;
   }
+
+  public static class PerRegionStatsImpl implements PerRegionStats {
```

Comment:
should we override toString and this class so java api users can easily print it out?
also on the owner class RebalanceResultImpl

Reply:
added toString